.travis.yml

```diff
@@ -60,7 +60,7 @@ before_script:
   - cd openFEC
   - git checkout $FEC_BRANCH
   - psql -c 'create database cfdm_test' -U postgres
-  - psql -f data/subset.sql cfdm_test -U postgres
+  - pg_restore --dbname cfdm_test data/subset.dump --no-acl --no-owner
```

Comment:
Yep, we deleted a bunch of old code from the API recently, which included the old subset. This code uses the current subset.